.thoth.yaml

```diff
@@ -16,6 +16,7 @@ managers:
     maintainers:
     - goern
     - fridex
+    - mayaCostantini
```

Comment:
I believe it would be better to just remove the `maintainers` section from `.thoth.yaml`. Kebechet should then pick up the list of maintainers from [the OWNERS file](https://github.com/thoth-station/prescriptions/blob/f4e47603df622e7ae6d58a7cecceb26b9d77a347/OWNERS#L3):

```suggestion
```